apps/work/views.py

```diff
@@ -31,6 +32,43 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context["tags"] = Tag.objects.all()
         context["work_types"] = WorkType.objects.all()
+        work_with_cover_description = Work.objects.exclude(
+            Q(cover_description__isnull=True) | Q(cover_description__exact="")
+        )
+        work_with_no_cover_description = Work.objects.filter(
+            Q(cover_description__exact="")
```

Comment:
```suggestion
            Q(cover_description__isnull=True) | Q(cover_description__exact="")
```
Are we ignore `null`?